standard/expressions.md

```diff
@@ -1283,6 +1285,50 @@ In a member access of the form `E.I`, if `E` is a single identifier, and if the
 > ```
 > Within the `A` class, those occurrences of the Color identifier that reference the Color type are delimited by `**`, and those that reference the Color field are not. *end example*
 
+### §null-conditional-member-access Null Conditional Member Access
+
+A *null_conditional_member_access* is a conditional version of *member_access* ([§12.7.5](expressions.md#1275-member-access)) and  it is a binding time error if the result type is `void`. For a null conditional expression where the result type may be `void` see (§null-conditional-invocation-expression).
+
+A *null_conditional_member_access* consists of a *primary_expression* followed by the two tokens "`?`" and "`.`", followed by an *Identifier* with an optional *type_argument_list*, followed by zero or more *dependent_access*es.
+
+```ANTLR
+null_conditional_member_access
+    : primary_expression '?' '.' Identifier type_argument_list? dependent_access*
+    ;
+    
+dependent_access
```

Comment:
Just to check, `primary_expression` will cover nested null-conditional member accesses, yes? So if we have:

```csharp
x?.M1()?.M2().M3
```

that ends up with a "top-level" null_conditional_member_access where the primary_expression is `x?.M1()?.M2()`, which is itself a null_conditional_member_access with a primary_expression of `x?.M1()`, which is itself a null_conditional_member_access with a primary expression of `x`... is that right?